### ISSUE TITLE
Make Live Snippet working with multiple cursors

### ISSUE THREAD

**Issue by basilgass** [tecosaur/LaTeX-Utilities#121]
## Feature Request
It would be nice to have snippet triggers also when using multi cursors.

### Is your feature request related to a problem? Please describe.
No.

### Describe the solution you'd like
Create multiple cursor.
When writing, for example xhat , all cursors should trigger the snippet.

### Describe alternatives you've considered
No alternative... except copy paste afterwards.

**Comment by tecosaur** [comment]
I agree, this would be great! I don't think this would require much more code, so much as careful thinking about the implementation.

I'll toss it on my to-do list for now :)
